8-external-github-webhook/code/src/functions/github_handler/index.ts

```diff
@@ -0,0 +1,45 @@
+import { client, betaSDK } from '@devrev/typescript-sdk';
+
+// Handles the event from GitHub
+async function handleEvent(event: any) {
+  // Extract necessary information from the event
+  const token = event.context.secrets.service_account_token;
+  const endpoint = event.execution_metadata.devrev_endpoint;
+
+  // Set up the DevRev SDK with the extracted information
+  const devrevSDK = client.setup({
+    endpoint: endpoint,
+    token: token,
+  });
+
+  // Extract the part ID and commits from the event
+  const partID = event.input_data.global_values.part_id;
+  const commits = event.payload.commits;
+
+  // Iterate through commits and append the commit message to the body of the comment
+  let bodyComment = '';
```

Comment:
Maybe add commiter and repo details at the start of the message.

Reply:
Will add - `Commits from GitHub:\n`